docs/usage/handheld/miyoocfw.md

```diff
@@ -0,0 +1,63 @@
+# MiyooCFW
+
+[MiyooCFW](https://github.com/TriForceX/MiyooCFW/wiki) ([Code](https://github.com/TriForceX/MiyooCFW)) is a custom firmware  for BittBoy, PocketGo, PowKiddy V90-Q90-Q20 and third party handheld consoles. It is based on the buildroot build environment and loosely based on OpenDingux. While it is intended for the named less powerful handhelds, it packs a good and wide selection of emulators. Some tinkering with the BIOS files is required though, but no worries, most of that is covered below or in their [Wiki](https://github.com/TriForceX/MiyooCFW/wiki/Emulator-Info)
+
+## BIOS Files
+
+MiyooCFW doesn't seem to have a centralized folder for putting BIOS files so it's a chore to prepare the card for running the provided emulators. Here's a try to help with finding the right ones and putting them in the right place.
+
+| System   | Emulator | Folder | Filenames | MD5SUMs | Comments |
```

Comment:
I'm 50/50 about including the table, knowing that the documentation here is likely to go stale as MiyooCFW development continues. But their wiki is also not the easiest to parse. Maybe we can caveat with "this list isn't guaranteed to be up to date, see the official wiki for more information."

Reply:
Thats a good idea. I will add that caveat and hipe you are ok with it since I think that having the BIOS focused interpretation of their table is worth it.

Reply:
https://github.com/emmercm/igir/pull/811/commits/4a675a97c855c0844a302f49e703edbd1e1e0b53 hopefully addresses this as you requested.

Reply:
That addition is great, thank you!

If you ever want to get fancy in the future, admonitions are great for calling out information like this: https://squidfunk.github.io/mkdocs-material/reference/admonitions